### PR TITLE
Users/t anmah/pylint import error fix

### PR DESCRIPTION
**Pull request by andreamah**
# Description:

Making sure that pylint doesn't flag the adafruit_circuitplayground library import as not found. This is done by suppressing the import-error in pylint for the workspace. Since pylint is often used in vscode (and users of the Python library are prompted to install it), we believed that this would be worth it.

## Type of change

- [ x ] Bug fix (non-breaking change which fixes an issue)

# Limitations:

Configuration is not updated until extension is opened for the first time (after compilation) 

# Testing:
- [ ] Testing with/without existing configurations in settings.json

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by vandyliu** [comment]
In the readme and how-to-use.md, there is 
```
- If you have pylint enabled, it might underline the import of the adafruit_circuitplayground library, but it will work correctly.
```
Can we remove that in this PR?

File: src/extension.ts [hunk 1]
```diff
@@ -880,21 +884,30 @@ const checkForTelemetry = (sensorState: any) => {
...
-  const pathToLib: string = __dirname;
-  const currentExtraPaths: string[] =
-    vscode.workspace.getConfiguration().get("python.autoComplete.extraPaths") ||
-    [];
-  if (!currentExtraPaths.includes(pathToLib)) {
-    currentExtraPaths.push(pathToLib);
+  updateConfigLists("python.autoComplete.extraPaths",[__dirname], vscode.ConfigurationTarget.Global)
+};
+
+const updatePylintArgs = () => {
```
Comment (isadorasophia) [inline src/extension.ts:890]: As we talked offline, I think it's worth investigating whether we can enable the adafruit import through pylint, so we don't have to disable these warnings. It looks like there is a [init-hook](https://stackoverflow.com/a/44732848) workaround in order to append a new path?

File: src/extension.ts [hunk 2]
```diff
@@ -880,21 +884,30 @@ const checkForTelemetry = (sensorState: any) => {
...
+  updateConfigLists("python.linting.pylintArgs",["--disable=E0401"], vscode.ConfigurationTarget.Workspace)
```
Comment (isadorasophia) [inline src/extension.ts:891]: What happens if pylint is not installed?

File: src/extension.ts [hunk 3]
```diff
@@ -880,21 +884,30 @@ const checkForTelemetry = (sensorState: any) => {
...
+}
+
+const updateConfigLists = (section: string, newItems: string[], scope: vscode.ConfigurationTarget) => {
+  // function for adding elements to configuration arrays
+  const currentExtraItems: string[] = vscode.workspace.getConfiguration().get(section) || [];
+
+  for (const item of newItems) {
+    if (!currentExtraItems.includes(item)) {
```
Comment (isadorasophia) [inline src/extension.ts:899]: You can also use a python `set()` instead

**Review by isadorasophia — CHANGES_REQUESTED** [review]
that seems like a very confusing issue for inexperienced devs, nice job handling it! 🌟

File: src/extension.ts (hunk 1 above)
Comment (andreamah) [reply src/extension.ts:890]: I looked into init-hook and am trying to debug the issues :0. Looking into it, I believe that the previous group tried to add the python_libs to the system path already in process_user_code.py (~lines 15-18). However, when checking the included modules in pylint (using `pylint --init-hook 'import sys; print(sys.modules.keys())'`) , the libraries aren't there :(. I'll try to look more into it!

**Comment by andreamah** [comment]
Seems a bit too high-effort for the importance. Will re-do later if necessary.

**Pull request closed without merging**